recipe/conda_build_config.yaml

```diff
@@ -8,6 +8,8 @@ c_compiler_version:            # [unix]
   - 16                         # [osx]
   - 10                         # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux]
   - 11                         # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and linux]
+  - 12                         # [os.environ.get("cf_cuda_enabled", "false") == "true" and linux]
+  - 12                         # [os.environ.get("cf_cuda_enabled", "false") == "true" and linux]
```

Comment:
This is incorrectly lower-cased here (that environment variable is not set by smithy)